packages/jaeger-ui-components/src/common/TraceName.tsx

```diff
@@ -42,7 +48,7 @@ export default function TraceName(props: Props) {
   const { className, error, state, traceName } = props;
   const isErred = state === fetchedState.ERROR;
   let title: string | React.ReactNode = traceName || FALLBACK_TRACE_NAME;
-  const styles = getStyles();
+  const styles = getStyles(useTheme());
```

Comment:
I'd recommend renaming this to `useTraceTheme`, to immediately make it clear that this is not the general grafana `useTheme`.

Reply:
Again because this is treated as a separate library this still does not try to assume much about grafana and be generic enough so it could be used elsewhere.

Reply:
Alright, thanks. :)